low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -143,14 +165,18 @@ def validate(self):
                              "%s - Argument validation failure. %s",
                              self.name,
                              "Post install requires global config.")
-        self.global_config = self.args.config[0]
-        Conf.load('global_config', self.global_config)
-        product = Conf.get('global_config', 'release>product')
+
+        # Dump global config
+        prvsnr_global_config_url = self.args.config[0]
```

Comment:
shouldn't this global cfg dump should be in init(), why in validate here ?

Reply:
Now I moved config dump method to init method of post_install.